clipping_manager/templates/clipping_manager/browse.html

```diff
@@ -52,5 +53,8 @@ <h1>{% trans "Your Clippings" %}</h1>
     </div>
 
     {% include "partials/pagination.html" %}
+{% endblock %}
 
-{% endblock %}
\ No newline at end of file
+{% block additional_js %}
```

Comment:
Fancy, didn't use the onload like this before. (I often either simply included on load listeners in JavaScript or just directly put the js in the template (which seems odd, but with sekizai tags it's actually quite nice, because one can put small js logic close to the template it belongs to (eg the list item(!) template and sekizai tags squashes them to large inline js (which obviously also has some disadvantages))

Reply:
The strong need to separate everything frontend related forced me here to create an extra file, but in such a small feature these _sekizai tags_ seem also reasonable